tests/fixtures/controllers/exampleController.ts

```diff
@@ -126,4 +126,20 @@ export class ExampleTestController {
     res?.(400, 123, { 'custom-header': 'hello' });
     return 'test 1';
   }
+
+  /**
+   * @param res The alternate response
+   * @example res.NoSuchCountry { "errorMessage":"No such country", "errorCode": 40000 }
```

Comment:
Do we need the `.` separator at all?

Looking at something like
```ts
   @example res NoSuchCountry { "errorMessage":"No such country", "errorCode": 40000 }
```

Is there any ambiguity if we split on `' '` and determine based on the length of the result?